ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_path_selection.schema.yml

```
@@ -0,0 +1,180 @@
+# Copyright (c) 2023 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  router_path_selection:
+    type: dict
+    description: Dynamic path selection configuration
+    keys:
+      peer_dynamic_source:
+        type: str
+        valid_values:
+          - stun
+        description: Source of dynamic peer discover. Only 'stun' is supported.
+      path_groups:
+        type: list
+        primary_key: name
+        items:
+          type: dict
+          keys:
+            name:
+              type: str
+              description: Path group name
+            id:
+              type: int
+              description: Path group ID
+              convert_types:
+                - str
+              min: 1
+              max: 65535
+            ipsec_profile:
+              type: str
+              description: IPSec profile for the path group
+            flow_assignment:
+              type: str
+              valid_values:
+                - lan
+              description: |
+                - Flow assignement LAN can not be configured in a path group with dynamic peer
+            local_interfaces:
+              type: list
+              primary_key: name
+              items:
+                type: dict
+                keys:
+                  name:
+                    type: str
+                    description: |
+                      Local interface name
+
+                      Ethernet<X> is supported
```

No need to repeat the pattern.
```suggestion
                      Local interface name.
```